scipy/stats/_entropy.py

```diff
@@ -46,20 +47,20 @@ def entropy(pk: np.typing.ArrayLike,
 
     Returns
     -------
-    S : {float, array_like}
+    H : {float, array_like}
         The calculated entropy.
 
     Notes
     -----
     Informally, the Shannon entropy quantifies the expected uncertainty
     inherent in the possible outcomes of a discrete random variable.
-
-    If messages consisting of sequences of symbols from a set S are to be
+    For example,
+    if messages consisting of sequences of symbols from a set ``S`` are to be
```

Comment:
```suggestion
    if messages consisting of sequences of symbols from a set are to be
```
name is not used again, so let's keep `S` as the name of the return above and omit this.